src/components/MessageBox.js

```diff
@@ -0,0 +1,96 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    expandable: React.PropTypes.bool,
```

Comment:
After looking at the design docs on this I think we can get rid of this `expandable` prop and assume that it's expandable if there are children. And also add the `isOpen` prop that I mentioned before.